continuous-profiling.graphqls

```diff
@@ -0,0 +1,84 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Save service policy
+input ContinuousProfilingServicePolicySaveRequest {
+    # service of the policy
+    serviceId: ID!
+    # target of the policy
+    targets: [ContinuousProfilingServicePolicyTargetSaveRequest!]!
+}
+
+input ContinuousProfilingServicePolicyTargetSaveRequest {
+    targetType: ContinuousProfilingTargetType!
+    checkItems: [ContinuousProfilingServicePolicyItemSaveRequest!]!
+}
+
+# Policy item of continuous profiling
+input ContinuousProfilingServicePolicyItemSaveRequest {
+    # define the monitor type to collect metrics
+    type: ContinuousProfilingMonitorType!
+    # threshold of policy, which decide by the monitor type
+    threshold: String!
+    # the length of time to evaluate the metrics
+    period: Int!
+    # how many times after the metrics match the threshold, will trigger profiling
+    count: Int!
+    # the URI path/regex filter when monitor the HTTP related types
+    uriList: [String!]
+    uriRegex: String
+}
+
+enum ContinuousProfilingTargetType {
+    # eBPF On CPU Profiling
+    ON_CPU,
+    # eBPF Off CPU Profiling
+    OFF_CPU,
+    # eBPF Network Profiling
+    NETWORK
+}
+
+type ContinuousProfilingSaveResult {
+    # TRUE if the policy is created successfully
+    status: Boolean!
+    # error reason when status == FALSE
+    errorReason: String
+}
+
+type ContinuousProfilingServicePolicyTarget {
+    targetType: ContinuousProfilingTargetType!
+    checkItems: [ContinuousProfilingServicePolicyItem!]!
+}
+
+# Similar with ContinuousProfilingServicePolicyItemSaveRequest
```

Comment:
What does this `Similar` mean?

Reply:
Because of GraphQL type restrictions(type/input), these two data structures are identical

Reply:
You could put this as a PR comment to remind reviewers, but not in the protocol/code. It would be confused when ppl read this from contributor perspective.

Reply:
updated, please re-check.